contracts/nois-gateway/src/state.rs

```diff
@@ -1,12 +1,16 @@
 use cosmwasm_schema::cw_serde;
-use cosmwasm_std::{Addr, Binary, StdResult, Storage};
+use cosmwasm_std::{Addr, Binary, Coin, StdResult, Storage};
 use cw_storage_plus::{Deque, Item, Map};
 
 #[cw_serde]
 pub struct Config {
     /// The address of the drand contract.
     /// As long as this is unset, noone can submit randomness.
     pub drand: Option<Addr>,
+    /// Manager to set the price and drand address
+    pub manager: Addr,
+    /// The price to pay in order to register the randomness job
+    pub prices: Coin,
```

Comment:
Let's change this to singular if we only have one option

```suggestion
    pub price: Coin,
```